doc/api/fs.md

```diff
@@ -2762,8 +2762,6 @@ added: v0.1.31
 
 * `filename` {string|Buffer}
 * `listener` {Function|undefined} **Default:** `undefined`
```

Comment:
Since `listener` is listed as optional in the signature, we may just remove the `undefined` here and mention it is optional.

Also to avoid the confusion, maybe something like

```
- `listener` {Function} Optional, a listener previously attached using `fs.watchFile()`
```

?

Reply:
Updated.